### PR TITLE
[SLS-1525] Add runtime & memorysize tags to enhanced metrics

### PR DESCRIPTION
**Pull request by hghotra**
### What does this PR do?

Add runtime & memorysize tags to enhanced metrics emitted by the Datadog Lambda Extension (used by the Serverless Detail page).

image

### Motivation

https://datadoghq.atlassian.net/browse/SLS-1525

### Additional Notes

Anything else we should know when reviewing?

### Describe how to test your changes

Write here in detail how you have tested your changes
and instructions on how this should be tested in QA.

Describe or link instructions to set up environment 
for testing, if the process requires more than just
running the agent on one of the supported platforms.

### Checklist

- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] The `need-change/operator` and `need-change/helm` labels has been applied if applicable.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated if applicable.

Note: Adding GitHub labels is only possible for contributors with write access.

### CODE REVIEW

File: pkg/serverless/tags/tags.go [hunk 1]
```diff
@@ -47,6 +51,11 @@ func BuildTagMap(arn string, configTags []string) map[string]string {
 	tags = setIfNotEmpty(tags, versionKey, os.Getenv(versionEnvVar))
 	tags = setIfNotEmpty(tags, serviceKey, os.Getenv(serviceEnvVar))
 
+	cleanedRuntime := strings.Replace(os.Getenv(runtimeVar), "AWS_Lambda_", "", 1)
+
+	tags = setIfNotEmpty(tags, runtimeKey, os.Getenv(cleanedRuntime))
```
Comment (maxday) [inline pkg/serverless/tags/tags.go:56]: I think the value you're looking for is in `cleanedRuntime` not in os.Getenv(cleanedRuntime)` ?

File: pkg/serverless/tags/tags.go [hunk 2]
```diff
@@ -32,6 +34,8 @@ const (
 	envKey                   = "env"
 	versionKey               = "version"
 	serviceKey               = "service"
+	runtimeKey               = "runtime"
+	memorySizeKey            = "memorysize"
```
Comment (maxday) [inline pkg/serverless/tags/tags.go:38]: Can you add a unit test for this?

Comment (hghotra) [reply pkg/serverless/tags/tags.go:38]: ✅

**Review by maxday — APPROVED** [review]
LGTM ! 💯

**Pull request merged**